### DISCUSSION TITLE
Could Consensys host a trust setup ceremony to build known curves SRS, such as BN254/BLS12377+BW6761/ BLS24-315+BW6-672/BLS12381?

### DISCUSSION THREAD

**Discussion by readygo67** [Consensys/gnark#1014] [General]
This SRS will be public goods for zk-snark.

**Comment by yelhousni** [comment]
Currently (or as part of its roadmap) Consensys, for the Linea zkEVM, uses transcripts from public trusted setup ceremonies. For:
- BN254: Ignition ceremony from Aztec,
- BLS12-377: ceremony from Aleo,
- BW6-761: Plumo ceremony from Celo.

There are setups for BLS12-381 but their use is not part of Consensys roadmap. For BLS24-315 and BW6-762, indeed I don't believe there are corresponding trusted public setups. `gnark` has already all the tools to conduct MPC ceremonies https://github.com/Consensys/gnark/pull/515 (contribution from Binance @HSG88) following [[BG19]](https://eprint.iacr.org/2017/1050.pdf) for all supported curves in gnark-crytpo https://github.com/Consensys/gnark/pull/563. Also some independent teams, for the Ethereum KZG ceremony, used gnark to implement a kzg CLI interface https://github.com/jsign/go-kzg-ceremony-client and a browser interface https://www.dogekzg.com/. We're happy to help if you want to take the lead on this SRS project.

**Reply by readygo67** [reply]
@yelhousni  Do you know where can I find the  BW6-761 SRS generated in  Plumo ceremony? I try to search in google but found nothing.

**Reply by yelhousni** [reply]
You can find the generated transcript (for phase1 aka powers of tau) here: https://console.cloud.google.com/storage/browser/plumoceremonyphase1
